File: Python/Mahasiswa.py
# import kelas Sivitas Akademik
from SivitasAkademik import SivitasAkademik

# deklarasi kelas Mahasiswa yang merupakan child dari kelas Sivitas Akademik
class Mahasiswa(SivitasAkademik):

    # atribut2 privat milik kelas Mahasiswa
    __nim = 0
    __prodi = ""
    __fakultas = ""

    # constructor
    def __init__(
        self,
        nik=0,
        nama="",
        jenis_kelamin="",
        asal_universitas="",
        email_edu="",
        nim=0,
        prodi="",
        fakultas="",
    ):
        # constructor super untuk kelas Sivitas Akademik
        super().__init__(nik, nama, jenis_kelamin, asal_universitas, email_edu)
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # setter nim
    def setNim(self, nim):
        self.__nim = nim

    # setter prodi
    def setProdi(self, prodi):
        self.__prodi = prodi

    # setter fakultas
    def setFakultas(self, fakultas):
        self.__fakultas = fakultas

    # getter nim
    def getNim(self):
        return self.__nim

    # getter prodi
    def getProdi(self):
        return self.__prodi

    # getter fakultas
    def getFakultas(self):
        return self.__fakultas

File: PHP/Mahasiswa.php
<?php
require("SivitasAkademik.php");

// deklarasi kelas Mahasiswa yang merupakan child dari Sivitas Akademik
class Mahasiswa extends SivitasAkademik
{
    // atribut2 milik kelas Mahasiswa
    private $nim;
    private $prodi;
    private $fakultas;

    // method2 milik kelas Mahasiswa

    // constructor
    public function __construct($nik, $nama, $jenis_kelamin, $asal_universitas, $email_edu, $nim, $prodi, $fakultas)
    {
        // constructor super untuk kelas Sivitas Akademik
        parent::__construct($nik, $nama, $jenis_kelamin, $asal_universitas, $email_edu);
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    // setter nim
    public function setNim($nim)
    {
        $this->nim = $nim;
    }

    // getter prodi
    public function setProdi($prodi)
    {
        $this->prodi = $prodi;
    }

    // setter fakultas
    public function setFakultas($fakultas)
    {
        $this->fakultas = $fakultas;
    }

    // getter nim
    public function getNim()
    {
        return $this->nim;
    }

    // getter prodi
    public function getProdi()
    {
        return $this->prodi;
    }

    // getter fakultas
    public function getFakultas()
    {
        return $this->fakultas;
    }


}

?>